sycl/test/check_device_code/bitfield_accesses.cpp

```diff
@@ -0,0 +1,44 @@
+// RUN: %clangxx -O0 -fsycl -fsycl-targets=spir64 %s -S -emit-llvm -o- | FileCheck -check-prefix=CHECK-FINE %s
```

Comment:
Don't we have sycl tests under `clang/tests` with some mock `sycl/sycl.hpp` ?

Reply:
May be I can move this to clang/test/Driver and prefix its name with sycl-?
Is that agreeable? I see sycl.hpp file included using '#include "sycl.hpp"'. Think that should work.

Thanks

Reply:
Ok..Moved it..thanks